src/Microsoft.AspNetCore.Testing/xunit/OperatingSystems.cs

```diff
@@ -1,12 +1,15 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using System;
+
 namespace Microsoft.AspNetCore.Testing.xunit
 {
+    [Flags]
     public enum OperatingSystems
     {
-        Linux,
-        MacOSX,
-        Windows
+        Linux = 1,
+        MacOSX = 2,
```

Comment:
If you want to be a terrible person, you could add an option called `XPlat = Linux | MacOSX`.

Reply:
I don't think that it's useful to add this and I am not a terrible person (I like to believe so). cc @Eilon

Reply:
Yeah let's not be bad. I think adding XPlat would be actually bad because it has a weak definition, meaning its definition could change in the future, and that could break tests.